contrib_bots/test-bots

```diff
@@ -17,11 +18,11 @@ def dir_join(dir1, dir2):
 if __name__ == '__main__':
     description = 'Script to run test_<bot>.py files in bots/<bot> directories'
     parser = argparse.ArgumentParser(description=description)
-    parser.add_argument('--bot',
+    parser.add_argument('--only',
                         nargs=1,
                         type=str,
                         action='store',
-                        help='test specified single bot')
+                        help='specified single bot (or comma-separated list of bots)')
```

Comment:
Updated to take this into account; I just extended what was already there, previously.

Ideally using the 'choices' option of argparse would seem logical and easier than the explicit test further down, but I couldn't get it to work with no arguments (it complains of [] not being a valid option, or similar).